libnavigation-core/src/test/java/com/mapbox/navigation/core/replay/route2/ReplayRouteSmootherTest.kt

```diff
@@ -0,0 +1,158 @@
+package com.mapbox.navigation.core.replay.route2
+
+import com.mapbox.geojson.LineString
+import com.mapbox.geojson.Point
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertTrue
+import org.junit.Assert
+import org.junit.Test
+
+class ReplayRouteSmootherTest {
+
+    private val recommendedThresholdMeters = 3.0
+
+    private val routeSmoother = ReplayRouteSmoother()
+
+    @Test
+    fun `should turn straight road into a segment`() {
+        // Geometry used qnq_gAxdhmhFuvBlJe@?qC^
```

Comment:
Could we be more explicit here? Are the `coordinates` obtained from this geometry using Turf?

Reply:
same as below comment, just removing the geometry since the coordinates are there